### PR TITLE
[Flickr] [Test] [Features] Command

### PR DESCRIPTION
**Pull request by thongphp**

### CODE REVIEW

File: app/Console/Commands/Flickr/FlickrPhotoSizes.php [hunk 1]
```diff
@@ -30,16 +30,14 @@ final class FlickrPhotoSizes extends BaseCommand
...
     public function fully(): bool
     {
-        if (!$photos = app(PhotoRepository::class)->getPhotosWithNoSizes()) {
-            return true;
-        }
-
-        $this->output->note(sprintf('Working on %d photos', $photos->count()));
+        $photos = app(PhotoRepository::class)->getPhotosWithNoSizes();
 
         if ($photos->isEmpty()) {
```
Comment (jooservices) [inline app/Console/Commands/Flickr/FlickrPhotoSizes.php:35]: Show notice " There are no photos "

File: tests/Feature/Console/Command/Flickr/FlickrPhotosTest.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        $this->expectsJobs(FlickrContactPhotos::class);
+        $this->expectsJobs(FlickrContactFavouritePhotos::class);
+        $this->artisan('flickr:photos');
+    }
+
+    public function testExecuteGetSingleContactAndStartGetPhotosAndFavPhotosQueueWithResetStates(): void
+    {
+        $this->artisan('flickr:contacts')->assertExitCode(0);
+
+        FlickrContactModel::where([])->update([FlickrContactModel::KEY_PHOTO_STATE => 1]);
```
Comment (jooservices) [inline tests/Feature/Console/Command/Flickr/FlickrPhotosTest.php:39]: Why use where nothing ?

File: tests/Feature/Console/Command/Flickr/FlickrContactTest.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        $this->artisan('flickr:contacts')->assertExitCode(0);
+        $this->expectsJobs(FlickrContact::class);
+        $this->artisan('flickr:contact');
+    }
+
+    public function testExecuteGetSingleContactAndStartGetContactInfoQueueWithResetStates(): void
+    {
+        $this->artisan('flickr:contacts')->assertExitCode(0);
+
+        FlickrContactModel::where([])->update([FlickrContactModel::KEY_STATE => 1]);
```
Comment (jooservices) [inline tests/Feature/Console/Command/Flickr/FlickrContactTest.php:35]: Same. Why where nothing ???

File: app/Console/Commands/Flickr/FlickrPhotoSizes.php (hunk 1 above)
Comment (thongphp) [reply app/Console/Commands/Flickr/FlickrPhotoSizes.php:35]: Will update in next commit.

File: tests/Feature/Console/Command/Flickr/FlickrPhotosTest.php (hunk 2 above)
Comment (thongphp) [reply tests/Feature/Console/Command/Flickr/FlickrPhotosTest.php:38]: To get all contact, use "where" for populate an query builder to mass update data.

File: tests/Feature/Console/Command/Flickr/FlickrContactTest.php (hunk 3 above)
Comment (thongphp) [reply tests/Feature/Console/Command/Flickr/FlickrContactTest.php:35]: To get all contact, use "where" for populate an query builder to mass update data.

File: tests/Feature/Console/Command/Flickr/FlickrPhotosTest.php (hunk 2 above)
Comment (jooservices) [reply tests/Feature/Console/Command/Flickr/FlickrPhotosTest.php:38]: `Model::query()->update(['confirmed' => 1]);`

File: tests/Feature/Console/Command/Flickr/FlickrContactTest.php (hunk 3 above)
Comment (jooservices) [reply tests/Feature/Console/Command/Flickr/FlickrContactTest.php:35]: Same as above

**Pull request merged**